resources/config.edn

```diff
@@ -1,6 +1,6 @@
 {:secrets    #include #join [#env HOME "/.villagebook-secrets.edn"]
 
- :db-spec    #profile {:default    "jdbc:postgresql://localhost:5432/villagebook"
+ :db-spec    #profile {:default    "jdbc:postgresql://localhost:5432/villagebook?user=postgres"
                        :test       "jdbc:postgresql://localhost:5432/villagebook_test"
```

Comment:
The db host needs to be changed to `villagebook-postgres:5432` for connections to be made from `villagebook-backend`.

Reply:
@prabhanshuguptagit 

localhost should work right? because we are binding to 5432 port for host machine.
By hardcoding `villagebook-backend` in config file will make user to add host entry in hostmachine. I found that an extra dependency to user?

Correct me if I am wrong.

Reply:
Docker containers run on a separate, isolated network than the host's. Using `localhost` breaks on my machine (you can test this by signing up, an exception is thrown). One way to work around this would be to make the container use the host network: https://docs.docker.com/network/host

shadow-cljs's proxy is also breaking currently for the same reason (specified in `shadow-cljs.edn`), it also needs fixing.